build-binutils.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/bash
+
```

Comment:
```suggestion
set -e

```

and then can skip all the `&&` below.